modules/replicator/src/main/java/org/apache/ignite/internal/replicator/message/TableAware.java

```diff
@@ -20,9 +20,10 @@
 import org.apache.ignite.internal.network.NetworkMessage;
 
 /**
- * Generic interface for all messages about concrete table.
+ * This class represents a common base interface for all message types related to a particular table.
+ * Extending this interface means the message is propagated to the table replica to be processed via zone replica.
  */
 public interface TableAware extends NetworkMessage {
-    /** Table Id. */
+    /** Returns table identifier. **/
```

Comment:
`Returns an identifier of a table for which one the message is intended (related?)`?

Reply:
:ok_hand: